metals/src/main/scala/scala/meta/internal/metals/CodeActionProvider.scala

```diff
@@ -46,4 +47,19 @@ final class CodeActionProvider(
     Future.sequence(actions).map(_.flatten)
   }
 
+  def executeCommands(
+      codeActionCommandData: CodeActionCommandData,
+      token: CancelToken
+  )(implicit ec: ExecutionContext): Future[Option[CodeActionCommandResult]] = {
+    val basePF: PartialFunction[CodeActionCommandData, Future[
+      Option[CodeActionCommandResult]
+    ]] =
+      PartialFunction.empty
+    val pf =
```

Comment:
do you mean `pf` or `basePF`?
In case of `pf` it uses the CancelToken that passes to the CodeActions' executeCommand, so I don't think it can be a private value, rather a private definition, but that would not change anything
`basePF` makes sense to have it as a private value, so that is not recreated every time